realhf/impl/model/modules/mlp.py

```diff
@@ -268,15 +268,13 @@ def __init__(
             self.c_fc = ColumnParallelLinear(
                 hidden_dim,
                 intermediate_dim,
-                async_tensor_model_parallel_allreduce=not sequence_parallel,
```

Comment:
This parameter has been [removed](https://github.com/openpsi-project/ReaLHF/blob/main/realhf/impl/model/parallelism/model_parallel/modules.py#L756C5-L771C53) in the MoE PR. I found this bug because this line is not highlighted in the editor.